lib/rules/spaced-comment.js

```diff
@@ -49,6 +50,17 @@ module.exports = function(context) {
         markerMatcher = new RegExp("((^(" + markers.join("))|(^(") + ")))[ \\t\\n]");
     }
 
+    /**
+     * Check to see if the block comment is jsDoc comment
+     * @param {ASTNode} node comment node
+     * @returns {boolean} True if its jsdoc comment
+     * @private
+     */
+    function isJsdoc(node) {
+        // make sure comment is multi-line and it start with /**\n
+        return node.loc.end.line > node.loc.start.line && jsDocMatcher.test(node.value);
```

Comment:
No need to test of its multiline since JSDoc comments don't have to be multiline (though they always must be block comments)